### ISSUE TITLE
Android TypeError: Cannot read property 'Commands' of undefined

### ISSUE THREAD

**Issue by adorum** [dominicstop/react-native-ios-context-menu#29]
@dominicstop Based on docs, `ContextMenuView` should render regular View component on Android. My android app crash with an error `TypeError: Cannot read property 'Commands' of undefined` at the moment of `import {ContextMenuView} from 'react-native-ios-context-menu'`. I conditionally render the component which contains this import statement only for ios platform. 

Any suggestions?

**Comment by nandorojo** [comment]
Until it's fixed, you could re-export an empty view inside of your own file, such as: `context-menu.android.tsx`:

`context-menu.tsx`
```ts
export { ContextMenuView } from 'react-native-ios-context-menu'
```

`context-menu.android.tsx`
```ts
import { View } from 'react-native'

export const ContextMenuView = View
```

And then, in any file, just do:

```ts
import { ContextMenuView } from './context-menu'
```

**Comment by adorum** [comment]
Thanks @nandorojo. Worked like a charm! Just can't find out why its working on android by default (w/o your suggestion) in the example project of this repo. Anyway thanks :)

**Comment by dominicstop** [comment]
hi, thanks for creating this issue ✨

sorry it took so long @adorum, but i implemented @nandorojo recommendation in the latest [1.7.1 patch](https://github.com/dominicstop/react-native-ios-context-menu/releases/tag/v1.7.1) haha

i used to use `*.ios.js`/`*.android.js` in an older version of this library (i.e. [v1.3.0](https://github.com/dominicstop/react-native-ios-context-menu/tree/8c41179dc2e919019415ef71508ce37232cc9d17/src)), but i forgot to add it back during the typescript re-write (due to metro + typescript-related bugs).

hopefully the latest patch fixes this (unfortunately, i don't have enough space to install android studio, so i can't verify if this fixes the bug haha)

**Comment by dominicstop** [comment]
this has now been fixed in [v1.7.2 patch](https://github.com/dominicstop/react-native-ios-context-menu/releases/tag/v1.7.2) ✨
<br>
screenshot:
![Screen Shot 2022-01-02 at 2 59 10 PM](https://user-images.githubusercontent.com/18517029/147868691-6ee75f4f-87b1-4f7d-af79-14acd0974745.png)
<br>
please re-open this issue if this patch doesn't fix the issue haha
